### ISSUE TITLE
[Batch Label Data] Add more label data for generative artificial intelligence

### ISSUE THREAD

**Issue by PureNatural** [X-lab2017/open-digger#1380]
### Description

Description 
I want to add some labeled data into OpenDigger to help us for our community analysis.
 
Label: Artificial Intelligence - speech

Type: Tech-1

Repos:

- elevenlabs/elevenlabs-python
- resemble-ai/resemble-alexa
- ZohaibAhmed/ChatGPT-Voice
- babysplly/Wellsaidlabs
- coqui-ai/TTS
- yacineMTB/scribepod
- Plachtaa/VALL-E-X
- neonbjb/tortoise-tts
- suno-ai/bark
- huggingface/transformers
- MycroftAI/mycroft-core
- common-voice/common-voice
- pandorabots/rosie
- singnet/dev-portal
- RasaHQ/rasa
- uberduck-ai/uberduck-ml-dev
- explosion/spaCy
- jovotech/jovo-framework
- fastai/fastai
- scikit-learn/scikit-learn

Label: Artificial Intelligence - music

Type: Tech-1

Repos:

- Harmonai-org/sample-generator
- MubertAI/Mubert-Text-to-Music
- lucidrains/musiclm-pytorch
- Cortexelus/dadabots
- AbhilashPal/MuseNet

Label: Artificial Intelligence - video

Type: Tech-1

Repos:

- replicate/cog
- nateraw/stable-diffusion-videos
- lucidrains/musiclm-pytorch
- Cortexelus/dadabots
- AbhilashPal/MuseNet
